README.md

```diff
@@ -6,100 +6,42 @@
+- [tiff](http://www.libtiff.org/) v4.0.9: Tag Image File Format library.
+- [cfitsio](http://heasarc.gsfc.nasa.gov/fitsio/fitsio.html): v4.0.0: Library of `C` and `Fortran` subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format.
+- [yaml-cpp](https://github.com/jbeder/yaml-cpp) v0.6.3: YAML parser and emitter in `C++`.
 - [casacore](http://casacore.github.io/casacore/) - Optional - Needed to interface with measurement
+- [ONNXruntime](https://onnxruntime.ai/) v1.17.1 - Optional - a cross-platform runtime engine based on the Open Neural Network eXchange format.
   sets. The main **PURIFY** program requires this library (and its dependencies)
-- [spdlog](https://github.com/gabime/spdlog) v1.9.2: Optional - Logging library. Downloaded automatically by conan.
 - [Catch2](https://github.com/catchorg/Catch2) v2.13.9: Optional -  A `C++`
-  unit-testing framework only needed for testing. Downloaded automatically by conan.
+  unit-testing framework only needed for testing.
 - [google/benchmark](https://github.com/google/benchmark) v1.6.0: Optional - A `C++`
-  micro-benchmarking framework only needed for benchmarks. Downloaded automatically by conan.
+  micro-benchmarking framework only needed for benchmarks.
 
 
 ## Installing and building PURIFY
 
-**Using Conan v2 (recommended)**
-
-[Conan](https://docs.conan.io/en/latest/installation.html) is a C++ package manager that helps deal with most of the
-C++ dependencies as well as the **PURIFY** installation:
-
-1. Once the mandatory user-provided dependencies are present,
-   `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
-
-   ``` bash
-   git clone --recurse-submodules https://github.com/astro-informatics/purify.git
-   ```
-1. Create a `conan` package for `sopt`
-
-   ```bash
-   conan create /path/to/purify/sopt/ --build missing
-   ```
-   If you get an error about broken symlinks you can set `skip_broken_symlinks_check = True` in
-   your `~/.conan/conan.conf` file or
-   [set an environment variable](https://docs.conan.io/en/1.46/reference/env_vars.html#conan-skip-broken-symlinks-check)
-1. Then, the program can be built using `conan`:
-
-   ``` bash
-   cd /path/to/purify
-   mkdir build
-   cd build
-   conan install .. -of . --build missing
-   conan build .. -of .
-   ```
-
-You can turn the various options on and off by adding flags to the `conan install` command, e.g.
-
-  ```bash
-  conan install .. -of . --build missing -o cppflow=on -o openmp=on -o mpi=off
-  conan build .. -of .
-  ```
-
-The full list of build options can be found in the [conanfile](./conanfile.py).
-To install in directory `INSTALL_FOLDER`, add the following options to the conan build command:
-
-  ``` bash
-  conan build .. -of INSTALL_FOLDER
-  ```
-
-
-**Using CMake**
-
 If the dependencies are already available on your system, you can also install **PURIFY** manually like so
```

Comment:
Maybe add something like "For examples on how to install dependencies on Ubuntu and MacOS, see the cmake.yml file in .github/workflows". I can obviously add the spack install instructions here later